src/components/views/rooms/RoomSublist2.tsx

```diff
@@ -595,6 +595,12 @@ export default class RoomSublist2 extends React.Component<IProps, IState> {
         );
     }
 
+    private onScrollPrevent(e: React.UIEvent<HTMLDivElement>) {
+        // the RoomTile calls scrollIntoView and the browser may scroll a div we do not wish to be scrollable
+        // this fixes https://github.com/vector-im/riot-web/issues/14413
+        (e.target as HTMLDivElement).scrollTop = 0;
```

Comment:
I'm pretty sure this will break overscrolling on macs.

Reply:
What? Are the sublists scrollable on macs even with `overflow: hidden`?

Reply:
Overscrolling on macOS looks fine to me with this PR applied.